config/deploy.rb

```diff
@@ -0,0 +1,13 @@
+# config valid for current version and patch releases of Capistrano
+lock "~> 3.11.0"
+
+set :application, "redapp-server"
+set :repo_url, "https://github.com/resolvetosavelives/redapp-server.git"
+set :deploy_to, -> { "/home/deploy/apps/#{fetch(:application)}" }
+set :rbenv_ruby, '2.3.4'
```

Comment:
Let's use 2.5.1 instead. We should add a `.ruby-version` file with this as well.